core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -170,15 +170,15 @@ private[chisel3] trait HasId extends InstanceId {
     * @param defaultSeed Optionally provide default seed for computing the name
     * @return the name, if it can be computed
     */
-  private[chisel3] def _computeName(defaultPrefix: Option[String], defaultSeed: Option[String]): Option[String] = {
+  private[chisel3] def _computeName(defaultPrefix: Option[String], defaultSeed: Option[String], sep: String = "_"): Option[String] = {
```

Comment:
add scaladoc for sep

Reply:
The `sep` param is no longer necessary for this pr - should I remove it?

Reply:
Yeah that would be best.